src/tasks/block.rs

```diff
@@ -161,6 +173,27 @@ impl BlockBuilder {
         self.bundle_poller.evict();
     }
 
+    async fn filter_transactions(&mut self, in_progress: &mut InProgressBlock) {
+        // query the rollup node to see which transaction(s) have been included
+        let mut confirmed_transactions = Vec::new();
+        for transaction in in_progress.transactions.iter() {
+            let tx = self
+                .ru_provider
+                .get_transaction_by_hash(*transaction.tx_hash())
+                .await
+                .expect("failed to get receipt");
+            if tx.is_some() {
+                confirmed_transactions.push(transaction.clone());
+            }
+        }
+        tracing::info!(confirmed = confirmed_transactions.len(), "found confirmed transactions");
```

Comment:
this should be a trace too, imo

Reply:
https://github.com/init4tech/builder/pull/31/commits/b3d3d982707a5b87ec163b7e3a8e3d9c6dc0baf9